libs/accounts/src/lib/accounts-table.tsx

```diff
@@ -186,115 +204,113 @@ export const AccountTable = forwardRef<AgGridReact, AccountTableProps>(
             field="available"
             type="rightAligned"
             headerTooltip={t(
-              'This is the amount of collateral available in your general account.'
+              'Deposited on the network, but not allocated to a market. Free for use in new markets or orders.'
```

Comment:
What does "new markets" want to imply here? (And why is it contrasted to "orders"?)

Reply:
@neildawson here are some suggestions for the tooltips.